scripts/announce.py

```diff
@@ -67,6 +67,17 @@ def get_authors(revision_range):
     cur.discard('Homu')
     pre.discard('Homu')
 
+    # Update doc/source/names_wordlist.txt with the names of every author
+    names = []
+    [names.extend(re.sub('\W+', ' ', x).split()) for x in cur.union(pre)]
```

Comment:
Could use a generator expression instead of a list comprehension since you only iterate over `names` once